example/lib/util/locale/localization.dart

```diff
@@ -35,19 +46,30 @@ class Localization {
 
   String _t(String key, {List<dynamic>? args}) {
     try {
-      final value = _localisedValues[key] as String?; // ignore: avoid_as
-      if (value == null) return '$key';
-      if (args == null || args.isEmpty) return value;
-      var newValue = value;
-      // ignore: avoid_annotating_with_dynamic
-      args.asMap().forEach((index, dynamic arg) =>
-          newValue = _replaceWith(newValue, arg, index + 1));
-      return newValue;
+      final value = _localisedValues[key] as String?;
```

Comment:
This looks like sub-optimal code.
`final value = (_localisedOverrideValues[key] ?? _localisedValues[key]) as String?`